src/Lykke.Service.Dash.Job/Services/PeriodicalService.cs

```diff
@@ -45,14 +49,26 @@ public class PeriodicalService : IPeriodicalService
                 var tx = await _dashInsightClient.GetTx(item.Hash);
                 if (tx != null && tx.Confirmations >= _minConfirmations)
                 {
+                    var info = new
+                    {
+                        operationId = item.OperationId,
+                        amount = tx.GetAmount(),
+                        fees = tx.Fees,
+                        blockHeight = tx.BlockHeight
+                    }.ToJson();
```

Comment:
Optional. You don't need to do ```ToJson()``` if you will use ```WriteInfo``` extension method of ```ILog```

Reply:
1. What extension do you mean? I one I use accepts string for context param
2. I am also using the info param for here: 
_chaosKitty.Meow(info)
It accepts object and doing ToString() inside (right?) according to description. I will not get in this case what I want to see in log.

Reply:
1. this one https://github.com/LykkeCity/CommonDotNetLibraries/blob/master/src/Lykke.Common/Log/LogExtensions.cs#L31
2. operationId is enough for Meow().